src/Microsoft.Health.Fhir.Liquid.Converter/Filters/DateFilters.cs

```diff
@@ -72,6 +72,11 @@ public static string FormatAsDateTime(string input)
             }
         }
 
+        public static string Now(string input, string format = "yyyy-MM-ddTHH:mm:ss.FFFZ")
+        {
+            return DateTime.UtcNow.ToString(format);
```

Comment:
If `format` is null or empty, or other illegal format string, should we defense here or throw exception back to invoker?

Reply:
Good point. I checked the document of DateTime and null or empty format is OK. It will render a default format, which is short day with long time, like 2/3/2021 8:11:19 AM.
Besides, it defines some [standard DateTime formats](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings), like "d", "f", etc. These are valid format as well. If the format is invalid, it will throw a FormatException, which will be caught in render process. I've added more tests in the [unit tests](https://github.com/microsoft/FHIR-Converter/pull/195/files#diff-6ab74cb38943ae1f88fea0dceece2068184ca686d322c4e8c0ab8a1844e20785).

Reply:
Got it, thanks!